### PR TITLE
via adding --feature-gate option in config file to enable kubelet's I…

### PR DESCRIPTION
**Pull request by GeorgeGuo2018**
…Pv6DualStack feature gate

### CODE REVIEW

**Comment by jimangel** [comment]
@GeorgeGuo2018 thank you for the PR! I would recommend only fixing one language per PR so it does not conflict with other localization teams.

Each localization team has their own branching strategy that incorporates the changes from the English master branch for various Kubernetes versions.

Let me know if you have any questions!

/cc @tengqm @chenrui333

**Comment by GeorgeGuo2018** [comment]
> @GeorgeGuo2018 thank you for the PR! I would recommend only fixing one language per PR so it does not conflict with other localization teams.
> 
> Each localization team has their own branching strategy that incorporates the changes from the English master branch for various Kubernetes versions.
> 
> Let me know if you have any questions!
> 
> /cc @tengqm @chenrui333

Updated

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
 
 {{< caution >}}
-If you specify an IPv6 address block larger than a /24 via  `--cluster-cidr` on the command line, that assignment will fail.
+If you specify an IPv6 address block larger than a /64 via  `--cluster-cidr` on the command line, that assignment will fail.
+You can enable IPv6DualStack feature gate for kubelet via adding '--feature-gates="IPv6DualStack=true"' in kubelet's config file.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:63]: @GeorgeGuo2018 this extra line seems like information useful to readers, but it duplicates line 55.

Is it needed?

File: content/en/docs/concepts/services-networking/dual-stack.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
 
 {{< caution >}}
-If you specify an IPv6 address block larger than a /24 via  `--cluster-cidr` on the command line, that assignment will fail.
+If you specify an IPv6 address block larger than a /64 via  `--cluster-cidr` on the command line, that assignment will fail.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:62]: @GeorgeGuo2018 is this change technically accurate? I think the warning about blocks larger than `/24` is correct.

**Pull request closed without merging**